plenum/common/types.py

```diff
@@ -76,6 +78,7 @@ class f:  # provides a namespace for reusable field constants
 PLUGIN_TYPE_VERIFICATION = "VERIFICATION"
 PLUGIN_TYPE_PROCESSING = "PROCESSING"
 PLUGIN_TYPE_STATS_CONSUMER = "STATS_CONSUMER"
+PLUGIN_TYPE_AUTHENTICATOR = 'AUTHENTICATOR'
```

Comment:
Where do we use it? What is Authenticator plugin?

Reply:
Removed